frontend/packages/ux-editor/src/hooks/useSelectedFormLayoutSetName.ts

```diff
@@ -1,22 +1,25 @@
 import { useStudioUrlParams } from 'app-shared/hooks/useStudioUrlParams';
 import { useLayoutSetsQuery } from 'app-shared/hooks/queries/useLayoutSetsQuery';
-import { useReactiveLocalStorage } from 'app-shared/hooks/useReactiveLocalStorage';
+import { useEffect, useState } from 'react';
 
 export type UseSelectedFormLayoutSetNameResult = {
   selectedFormLayoutSetName: string;
   setSelectedFormLayoutSetName: (layoutName: string) => void;
-  removeSelectedFormLayoutSetName: () => void;
 };
 
 export const useSelectedFormLayoutSetName = (): UseSelectedFormLayoutSetNameResult => {
   const { org, app } = useStudioUrlParams();
   const { data: layoutSets } = useLayoutSetsQuery(org, app);
 
-  const [
-    selectedFormLayoutSetNameState,
-    setSelectedFormLayoutSetName,
-    removeSelectedFormLayoutSetName,
-  ] = useReactiveLocalStorage('layoutSet/' + app, undefined);
+  const savedLayoutSetName = localStorage.getItem('selectedFormLayoutSetName');
+  const [selectedFormLayoutSetNameState, setSelectedFormLayoutSetName] =
+    useState(savedLayoutSetName);
+
+  useEffect(() => {
+    if (selectedFormLayoutSetNameState) {
+      localStorage.setItem('selectedFormLayoutSetName', selectedFormLayoutSetNameState);
```

Comment:
What do you think of using the `typedLocalStorage` API we have built in our repo? This helps us parse/stringify and gives us some typing.😊 

```suggestion
      typedLocalStorage.setItem<string>(storageKey, selectedFormLayoutSetNameState);
```

Reply:
fixed